api_service.py

```
@@ -270,6 +285,54 @@ def validate_environment():
+            repository=repository,
+            buffer_size=CATEGORY_AGGREGATION_BUFFER_SIZE
+        )
+
+        # Initialize cleanup service
+        tally_cleanup_service = TallyCleanupService(
+            repository=repository,
+            config=config
+        )
+
+        logger.info(f"✅ Category aggregation initialized (buffer size: {CATEGORY_AGGREGATION_BUFFER_SIZE})")
+        return category_aggregator
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)